sdks/python/apache_beam/testing/load_tests/load_test.py

```diff
@@ -0,0 +1,83 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+import json
+import logging
+import os
+import unittest
+
+from apache_beam.testing.load_tests.load_test_metrics_utils import MetricsReader
+from apache_beam.testing.test_pipeline import TestPipeline
+
+load_test_enabled = False
+if os.environ.get('LOAD_TEST_ENABLED') == 'true':
+  load_test_enabled = True
+
+
+class LoadTest(unittest.TestCase):
+  def parseTestPipelineOptions(self, options=None):
+    if not options:
+      options = self.input_options
+
+    return {
+        'numRecords': options.get('num_records'),
+        'keySizeBytes': options.get('key_size'),
+        'valueSizeBytes': options.get('value_size'),
+        'bundleSizeDistribution': {
+            'type': options.get(
+                'bundle_size_distribution_type', 'const'
+            ),
+            'param': options.get('bundle_size_distribution_param', 0)
+        },
+        'forceNumInitialBundles': options.get(
+            'force_initial_num_bundles', 0
+        )
+    }
+
+  @classmethod
+  @unittest.skipIf(not load_test_enabled, 'Enabled only for phrase triggering.')
+  def setUpClass(cls):
```

Comment:
I'm not sure if `setUpClass` is best choice. Why can't you just use `setUp` here?
I can imagine (correct me if I'm wrong) that there can be more tests added to suites extending `LoadTest` and then `setUpClass` is executed once, before whole suite while probably it should be executed before each test.

Reply:
You are right. I forgot about the case where more tests may appear. This solution makes it even more simple, because setUpClass was forcing to add here also the skip.